### ISSUE TITLE
Investigate how to pass attacks w/o using YAML file

### ISSUE THREAD

**Issue by thomas-moulard** [aws-robotics/ROS2-SecTest#1]

**Comment by ryanewel** [comment]
Looks like this isn't currently possible but there is a feature request to add it here: https://github.com/ros2/rcl/issues/460

**Comment by xabxx** [comment]
@ryanewel any updates?

**Comment by ryanewel** [comment]
I don't think so, we still don't have a good way to do this from the command line and that issue is still open. Maybe we can mark this as a feature request?
